app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/movielist/viewmodel/MovieListViewModel.kt

```diff
@@ -8,9 +8,8 @@ import com.example.marcinwisniewski.intivetrainingapp.movielist.model.MovieRepos
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.Disposable
 import io.reactivex.schedulers.Schedulers
-import javax.inject.Inject
 
-class MovieListViewModel @Inject constructor(private val movieRepository: MovieRepository) : ViewModel() {
```

Comment:
i am not sure this is ok but I get rid of injecting repository here - I am injecting it now on the ViewModel factory and then creating view Model with repository passed as a normal constructor field.